server.js

```diff
@@ -1,38 +1,170 @@
 import express from 'express'
-import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import dotenv from "dotenv"
```

Comment:
not sure if this could be removed after connecting to heroku or if one or the other is better ?

Reply:
dotenv package is injecting your process variables when you work on localhost.